packages/packages/commonwell-sdk/src/client/commonwell.ts

```diff
@@ -418,6 +420,22 @@ export class CommonWell {
     return personSearchRespSchema.parse(resp.data);
   }
 
+  /**
+   * Searches for a person based on person id.
+   * See: https://specification.commonwellalliance.org/services/patient-identity-and-linking/protocol-operations#8713-find-persons-matching-patient-demographics
+   *
+   * @param meta        Metadata about the request.
+   * @param personId   The person ID.
+   * @returns
+   */
+  async searchPersonByUri(meta: RequestMetadata, personId: string): Promise<Person> {
+    const headers = await this.buildQueryHeaders(meta);
+    const resp = await this.api.get(`${CommonWell.PERSON_ENDPOINT}/${personId}`, {
+      headers,
+    });
+    return personSchema.parse(resp.data);
+  }
```

Comment:
does this actually work? I was trying to find this in the docs, but I didn't see a GET person by ID endpoint

Reply:
Yea it actually does and it might work for all types. I went to postman and just added this url as a get request `https://integration.rest.api.commonwellalliance.org/v1/person/6f5338aa-d8e6-4199-bd5c-21879507b87c`

You can try it out just make sure you have a valid jwt and it should work. This approach could technically make things easier for fetching one type. W'ed have to test it out though because its not in the docs

Reply:
The link refers to the self.href in the links created